compiler/common-artifacts/exclude.lst

```diff
@@ -7,6 +7,7 @@
 ## TensorFlowLiteRecipes
 optimize(Add_STR_000) # STRING is not supported
 optimize(Add_STR_001) # STRING is not supported
+optimize(Net_Gather_SparseToDense_AddV2_000) # Constant folding is not generally supported
```

Comment:
@jongwonyang How about splits this PR Into two? 
This one line can be the one PR. 

https://github.com/Samsung/ONE/commits/master/compiler/common-artifacts 
(Other PR works on common-artifacts submitted like above)

Reply:
Great idea! I'll make the split :)

Reply:
Also, Since `exclude.lst` should be merged first, separate PR should be opened one by one.

Reply:
@zetwhite On second thought, this exclusion is no more needed because it's fixed in https://github.com/Samsung/ONE/issues/13939

Although, it still generates wrong shaped output, it's OK for now because
- the shape check is not performed in optimization test
- and the key reason of wrong shape inference is lack of constant folding
  - which means if the constant folding is implemented, my shape inference algorithm will generate correct shape automatically.

Reply:
@jongwonyang Before I heard that without excluding, optimizer test failed with log -`ERROR: Optimized graph is invalid`. Is there a no more build problem?

Reply:
> @jongwonyang I heard that without excluding, optimizer test failed with log -`ERROR: Optimized graph is invalid`. Is there a no more build problem?

Yes, it builds successfully now.

I think it was invalid because of something else...

Reply:
Oh I see. That's a good news :) 

> I think it was invalid because of something else... 

Maybe the draft was not rebased on the master branch..?

Reply:
> Maybe the draft was not rebased on the master branch..?

Yes, the previous draft didn't reflect [this fix](https://github.com/Samsung/ONE/issues/13939).

Current draft is rebased on the newest branch. PTAL :)